deployment/npm/build.ts

```diff
@@ -86,16 +87,17 @@ if (version == null) {
     "homepage": "https://github.com/dprint/dprint#readme",
     // for yarn berry (https://github.com/dprint/dprint/issues/686)
     "preferUnplugged": true,
+    "engines": {
+      "node": ">=18.19.1"
+    },
     "scripts": {
-      "postinstall": "node ./install.js",
+      "postinstall": "node ./postinstall.js",
```

Comment:
I think this now requires the post install step? Previously dprint would work even when someone did `npm install --ignore-scripts`.

Reply:
Yes, it is true. This is cost for performance improvement

Reply:
@dsherret Or try #840. #840 should work with ignoring scripts